packages/odie-client/src/components/message/sources.tsx

```diff
@@ -36,7 +36,7 @@ export const Sources = ( { message }: { message: Message } ) => {
 			];
 		}
 		return [];
-	}, [ message?.context?.sources, trackEvent ] );
+	}, [ message.context?.sources, message?.message_id, trackEvent ] );
```

Comment:
It is probably more correct to check the `message` existence in both.

Reply:
I removed that by mistake, thanks for catching.